extensions/azdata/src/typings/azdata-ext.d.ts

```diff
@@ -233,20 +235,20 @@ declare module 'azdata-ext' {
 	export interface IAzdataApi {
 		arc: {
 			dc: {
-				create(namespace: string, name: string, connectivityMode: string, resourceGroup: string, location: string, subscription: string, profileName?: string, storageClass?: string): Promise<AzdataOutput<void>>,
+				create(namespace: string, name: string, connectivityMode: string, resourceGroup: string, location: string, subscription: string, profileName?: string, storageClass?: string, additionalEnvVars?: { [key: string]: string }): Promise<AzdataOutput<void>>,
```

Comment:
These should use the AdditionalEnvVars type

Reply:
Done

Reply:
Done